examples/layer-browser/src/examples/core-layers.js

```diff
@@ -85,8 +85,6 @@ const GeoJsonLayerExample = {
     },
     getWidth: f => f.properties['stroke-width'],
     getElevation: f => 500,
-    widthScale: 10,
```

Comment:
@ibgreen This is where I got confused. This two props are props of the underlying path layer, and they are not defined in the defaultProps of GeoJsonLayer, and we didn't mentioned them in the API audit, so I assume that we don't want to expose it. 

We need to discuss again what props we would like to have for the GeoJsonLayer.